save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -51,11 +57,26 @@ class UserSettingsOrganizationsMenuView : UserSettingsView() {
                                 }
                             }
                             div {
-                                className = ClassName("mr-3")
-                                val role = state.userInfo?.name?.let {
-                                    organizationDto.userRoles[it]
-                                } ?: Role.NONE
-                                +role.formattedName
+                                className = ClassName("col-5 align-self-right d-flex align-items-center justify-content-end")
+                                val role = state.userInfo?.name?.let { organizationDto.userRoles[it] } ?: Role.NONE
```

Comment:
@sanyavertolet (as expert of frontend) -- what's better: use state or provide required props via `Props` to `FC`?